arch/arm/boot/dts/qcom/qcom-msm8974-sony-xperia-rhine-togari.dts

```diff
@@ -0,0 +1,41 @@
+// SPDX-License-Identifier: GPL-2.0
+#include "qcom-msm8974-sony-xperia-rhine.dtsi"
+
+/delete-node/ &touchscreen;
```

Comment:
Add a comment why this is needed e.g. `/* Togari uses a different touchscreen compared to other rhine devices */`